HACKING.md

```diff
@@ -53,7 +53,7 @@ See `dfx canister call --help` and [the documentation](https://sdk.dfinity.org/d
 The `dfx` executable can proxy queries to the canister. To view it, run the following and open the resulting link in your browser:
 
 ```bash
-echo "http://localhost:4943?canisterId=$(dfx canister id internet_identity)"
```

Comment:
Do you still have a global network config? I think it should work fine with the default port 4943. The switch to the subdomain based routing is good though. 👍

Reply:
Well, I suppose that is because port `8080` is configured in `/Users/llorencmuntaner/.config/dfx/networks.json`

Reply:
Yes, but I didn't touch that file, I assumed everybody has the same unless they change it. Or is it me the only one using that port?

Reply:
@przydatek had that issues as well, worth figuring out how 8080 got there but as Frederik said 4943 is the default so let's use that

Reply:
8080 is needed for `dfx nns install`.

https://dfinity.slack.com/archives/CGA566TPV/p1708008254336109?thread_ts=1708007865.314369&cid=CGA566TPV

Reply:
Done!